proposals/0441-formalize-language-mode-terminology.md

```diff
@@ -150,7 +151,7 @@ Package (  // Error: Ambiguous use of 'init'
 )
 ```
 
-This follows the same approach used by all past revisions of the Package `init` method.
+Adding the `@_disfavoredOverlaod` attribute allows the compiler to disambiguate between the two methods.
```

Comment:
This entire section is probably unnecessary now.  It only used to exist because you needed to justify why the old initializer had to be obsoleted.

Reply:
Section removed in 540b9c6c77a48fefd68ca248daba749b6ed22875